src/components/serviceProviderSearch/ServiceProviderSearch.js

```
@@ -0,0 +1,122 @@
+import React from "react";
+import ServicesCategoryService from "../../services/ServiceCategoryService";
+import mockCategory from "../../mock/data/service.providers.mock";
+
+export default class ServicesProviderSearch extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.serviceCategoryService = ServicesCategoryService.getInstance()
+        this.state = {
+            serviceCategory: {
+                title: '',
+                id: 1
+            }
+        }
+    }
+
+    componentDidMount() {
+        this.serviceCategoryService.findServiceCategoryById(this.props.match.params.id)
+            .then(serviceCategory => {
+                    this.setState({
+                        serviceCategory: serviceCategory
+                    });
+                    console.log(this.state.serviceCategory)
+                }
+            ).catch(err => {

```

I love the error catching (we need more of it in general), but I would really avoid doing this. Rendering mock data in response to an error being caught is going to be very confusing down the line. The mock data should only be used in the test files.